CMSgov/design-system
# [WNMGDS-914] Add component examples to CRA and CRA-typescript

## Summary
Adding every component as an example to create-react-app and create-react-app-typescript to make testing of changes to components easier.

### Added
Examples folder under Components folder for both CRA and CRA-typescript example.

### Fixed
When adding the Accordion component to create-react-app-typescript, there was an error that no Accordion type was exported from the design system, which is why you see the fixes in the design-system package included.For this reason, when testing the create-react-typescript section, you will have to update the package.json to point to the local design-system and not the actual design-system for this PR.

## How to test
- cd to `examples/create-react-app`, yarn install, yarn build, yarn start - You should see a new Examples tab with an example of every component
- cd to `examples/create-react-app-typescript` - You can try the same steps and you'll see an error that there is no exported Accordion from the type folder. Change the package.json to point to the local design system package. Then run `(cd ../.. && yarn install && yarn build) && rm -rf node_modules && yarn install && yarn start` and it should compile and you should see examples tab
